azure-pipelines.yml

```diff
@@ -173,18 +203,142 @@ stages:
 
 - template: azure-pipelines-e2e-tests-template.yml
   parameters:
-    versions:
-    - '2.3.0'
-    - '2.3.1'
-    - '2.3.2'
-    - '2.3.3'
-    - '2.3.4'
-    - '2.4.0'
-    - '2.4.1'
-    - '2.4.3'
-    - '2.4.4'
-    - '2.4.5'
-    - '2.4.6'
-    - '2.4.7'
-    - '3.0.0'
-    - '3.0.1'
+    backwardCompatibleRelease: $(backwardCompatibleRelease)
+    forwardCompatibleRelease: $(forwardCompatibleRelease)
+    tests:
+    - version: '2.3.0'
+      jobOptions:
+      - pool: 'Hosted VS2017'
+        testOptions: ''
+        backwardCompatibleTestOptions: $(backwardCompatibleTestOptions_Windows_2_3)
+        forwardCompatibleTestOptions: $(forwardCompatibleTestOptions_Windows_2_3)
```

Comment:
Can you add a comment that this is disabled on ubuntu and add a link to the issue?

Reply:
Comment added